README.md

```diff
@@ -15,13 +15,13 @@ I'm a recent computer science graduate passionate about machine learning and adv
 Project source code: [https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan)
 
 
-**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/master/linux_sql)]: Set up monitoring scripts for a linux cluster to track usage statistics for several nodes. These nodes are connected locally and the script will periodically populate a table with each node's current usage information. This project helped get acquainted with the linux filesystem hierarchy. It was apparent that virutal memory comprises process info, memory, CPU scheduling and other statistics. Logging memory statistics was a good exercise in understanding file paritions, which manage the storage space.
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_SiddarthKrishnan/tree/master/linux_sql)]: Set up monitoring scripts for a linux cluster to track usage statistics for several nodes. These nodes are connected locally and the script our periodically populates a table with each node's current usage information. This project helped get acquainted with the linux filesystem hierarchy. It was apparent that virutal memory comprises process info, memory, CPU scheduling and other statistics. Logging memory statistics was a good exercise in understanding file paritions, which manage the storage space on the OS.
```

Comment:
Developed a monitoring program using Bash scripts and PostgreSQL for a Linux cluster to track hardware usage statistics, such as CPU, RAM, and disk. Linux servers are connected locally, and the script periodically populates a table with each node's current usage information. ~This project helped me get acquainted with the Linux filesystem hierarchy.~ It was apparent that virtual memory comprises process info, memory, CPU scheduling and other statistics. Logging memory statistics was a good exercise in understanding file partitions, which manage the storage space on the OS.

Docker, SQL, crontab